src/radha/NestedSwitchCase/Assignment15/BankOperations.java

```diff
@@ -0,0 +1,159 @@
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI */
+
+package radha.NestedSwitchCase.Assignment15;
+
+public class BankOperations {
+	double iciciBal=5000, hdfcBal=4500, bobBal=9000, sbiBal=6000;
+	int iciciCredit=0, hdfcCredit=0, bobCredit=0, sbiCredit=0, iciciDebit=0, hdfcDebit=0, bobDebit=0, sbiDebit=0;
+	static int totalDebitCnt=0, totalCreditCnt=0;
+	
+	void operation(String bankName, String operation, double amount) {
+		bankName = bankName.toLowerCase();
+		operation = operation.toLowerCase();
+		switch(operation) {
+		case "debit":
+			switch(bankName) {
+			case "icici":
+				if(amount <= iciciBal) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.